public/modules/custom/grants_admin_applications/src/Form/ResendApplicationsForm.php

```diff
@@ -637,7 +637,13 @@ public function buildAttachments(AtvDocument $atvDocument, array &$form): void {
       $attOk = $this->areAttachmentsOk($events, $attachment, $attachmentInfo, $appEnv);
 
       $fieldInfo = $this->findByFilename($attachment, $attachmentInfo);
-      $fieldLabel = $this->extractFieldValue($fieldInfo, 'description');
+
+      // Sometimes fieldInfo is null so we need to check for that so the next method won't fail.
+      if (is_array($fieldInfo)) {
+        $fieldLabel = $this->extractFieldValue($fieldInfo, 'description');
```

Comment:
`extractFieldValue()` seems to return `?string` so you can just cast the value to string: `$fieldLabel = (string) $this->extractFieldValue($fieldInfo, 'description');`

Reply:
done.